sys/net/network_layer/ng_ipv6/ng_ipv6.c

```diff
@@ -0,0 +1,501 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+#include <errno.h>
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "byteorder.h"
+#include "cpu-conf.h"
+#include "kernel_types.h"
+#include "net/ng_netbase.h"
+#include "net/ng_protnum.h"
+#include "thread.h"
+#include "utlist.h"
+
+#include "net/ng_ipv6/nc.h"
+#include "net/ng_ipv6/netif.h"
+
+#include "net/ng_ipv6.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define _MAX_L2_ADDR_LEN    (8U)
```

Comment:
As soon as #2833 is merged we can. Apart from that it does not really hurt for now here either.